extensions/kafka/src/test/java/com/hazelcast/jet/kafka/impl/KafkaTestSupport.java

```diff
@@ -49,19 +48,21 @@
 
 import static com.hazelcast.test.HazelcastTestSupport.randomString;
 import static java.util.Collections.emptyMap;
+import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
 public abstract class KafkaTestSupport {
+    static final long KAFKA_MAX_BLOCK_MS = MINUTES.toMillis(10);
     protected Admin admin;
     protected KafkaProducer<Integer, String> producer;
     protected String brokerConnectionString;
     private KafkaProducer<String, String> stringStringProducer;
 
     public static KafkaTestSupport create() {
-        if (!DockerTestUtil.dockerEnabled() || OsHelper.isArmMac()) {
+        if (!DockerTestUtil.dockerEnabled()) {
             if (System.getProperties().containsKey("test.kafka.version")) {
                 throw new IllegalArgumentException("'test.kafka.version' system property requires docker and x86_64 CPU");
```

Comment:
- Could you leave a comment here that Kafka older than 7.2.0 will fail on Arm machines? 
- The exception message here needs to be changed as well. 
- `OsHelper#isArmMac` is not used and can be removed.